pkg/platform/runner/show.go

```diff
@@ -0,0 +1,63 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package runner
+
+import (
+	"github.com/elastic/cloud-sdk-go/pkg/api"
+	"github.com/elastic/cloud-sdk-go/pkg/client/platform_infrastructure"
+	"github.com/elastic/cloud-sdk-go/pkg/models"
+	multierror "github.com/hashicorp/go-multierror"
+
+	"github.com/elastic/ecctl/pkg/util"
+)
+
+// ShowParams is the set of parameters required for
+type ShowParams struct {
+	Params
+	ID string
+}
+
+// Validate checks the parameters
+func (params ShowParams) Validate() error {
+	var merr = new(multierror.Error)
+
+	if params.ID == "" {
+		merr = multierror.Append(merr, util.ErrIDCannotBeEmpty)
+	}
+
+	merr = multierror.Append(merr, params.Params.Validate())
+
+	return merr.ErrorOrNil()
+}
+
+// Show returns information about a specific runner
+func Show(params ShowParams) (*models.RunnerInfo, error) {
+	if err := params.Validate(); err != nil {
+		return nil, err
+	}
+
+	res, err := params.API.V1API.PlatformInfrastructure.GetRunner(
+		platform_infrastructure.NewGetRunnerParams().WithRunnerID(params.ID),
```

Comment:
I think this line is too long, I'd add a break:
```suggestion
		platform_infrastructure.NewGetRunnerParams().
			WithRunnerID(params.ID),
```